vpr/src/draw/draw_basic.cpp

```diff
@@ -1103,6 +1103,87 @@ void draw_crit_path(ezgl::renderer* g) {
     }
 }
 
+/**
+ * @brief Draw critical path elements.
+ * 
+ * This function draws critical path elements based on the provided timing paths
+ * and indexes map. It is primarily used in server mode, where items are drawn upon request.
+ */
+void draw_crit_path_elements(const std::vector<tatum::TimingPath>& paths, const std::map<std::size_t, std::set<std::size_t>>& indexes, ezgl::renderer* g)
+{
+    t_draw_state* draw_state = get_draw_state_vars();
+    static ezgl::color contour_color{0,0,0,40};
```

Comment:
resolved https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/2526/commits/e2e9ad09008b9f0a3f6a2d9bb756f0f770598ab9
i used static keyword to avoid variable allocation on each draw_crit_path_elements call. But this didn't give benefit with simple structure such as ezgl::color, so i replaced static with keyword const.